src/codegen.cpp

```diff
@@ -4502,6 +4502,7 @@ static Function *emit_function(jl_lambda_info_t *lam)
     int prevlno = -1;
     for(i=0; i < stmtslen; i++) {
         jl_value_t *stmt = jl_cellref(stmts,i);
+        /*
```

Comment:
Forgot about this. If Jeff approves the parser change then all LineNode-handling code can be deleted.